src/s3access.cpp

```diff
@@ -44,6 +44,12 @@ S3Access::~S3Access()
     }
 }
 
+s3_date S3Access::now_as_s3date(void) const {
```

Comment:
For the sake of consistency, could the opening method bracket be put on its own line?